lib/caasp.pm

```diff
@@ -236,6 +236,18 @@ sub script_retry {
     }
 }
 
+# Wrapper returning PIPESTATUS[0]
+sub script_run0 {
```

Comment:
I just wonder if we should put this in os-autoinst. Potentially as an option of script_run itself.

Reply:
I thought about it, I can move it in os-autoinst if you think it would be useful.
I would like it as script_run option, but I don't see clean way to add another function parameter.

Reply:
I think we have this thing used in at least 5 places. So I guess it worth it already imho.

Reply:
I will merge this and move script_run0 and script_assert0 to testapi in another PR.